rtrlib/rtr/packets.c

```diff
@@ -19,6 +19,7 @@
 
 #define TEMPORARY_PDU_STORE_INCREMENT_VALUE 100
 #define MAX_SUPPORTED_PDU_TYPE 10
+#define INTERVAL_OPTION -1
```

Comment:
this will not work, should be `static int interval_option = SOME_DEFAULT_VALUE`, and there should be a API call to set this parameter.